modules/ossm-about-adding-namespace.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/installing-ossm.adoc
+
+:_content-type: CONCEPT
+[id="ossm-about-adding-namespace_{context}"]
+= About adding projects to a service mesh
+
+A project contains services; however, the services are not available to the mesh unless you add the project to the service mesh.
+
+[NOTE]
+====
+In OpenShift Container Platform, a project is essentially a Kubernetes namespace with additional annotations, such as the range of user IDs that can be used in the project. Typically, the {product-title} web console uses the term project, and the CLI uses the term namespace, but the terms are essentially synonymous.
+====
+
+You can add projects to an existing service mesh using either the web console or the CLI. There are two methods to add a project to a mesh:
+
+* Specifying the project name in the `ServiceMeshMemberRoll` resource.
+
+* Creating the `ServiceMeshMember` resource in the project.
+
+.ServiceMeshMemberRoll method
+
+This is the simplest way to add a project to a service mesh. To add a project, specify the name of the project where your applications reside in the `spec.members` field of the `ServiceMeshMemberRoll` resource.
+
+[NOTE]
+====
+Adding projects using this method requires the user to have the `update servicemeshmemberrolls` and the `update pods` privileges in the project that is being added.
+====
+
+The `ServiceMeshMemberRoll` resource lives alongside the `ServiceMeshControlPlane` resource, and specifies the list of projects that should be controlled by the control plane. The control plane, which includes Istiod, Ingress and Egress Gateways, and additional components such as Kiali and Jaeger, must be deployed in a different namespace than the {SMProductShortName} Operators and the applications and services (which are part of the data plane).
```

Comment:
Maybe reword it for clarity:
```suggestion
A `ServiceMeshMemberRoll` resource specifies which projects are controlled by a `ServiceMeshControlPlane` resource. The control plane has Istiod, Ingress, and Egress Gateways, and other components like Kiali and Jaeger. The control plane must be in a separate namespace from the {SMProductShortName} Operators and the data plane applications and services.
```
This suggestion probably needs SME review.

Reply:
Done